custom-elements-manifest.config.mjs

```diff
@@ -0,0 +1,14 @@
+import { moduleFileExtensionsPlugin } from 'cem-plugin-module-file-extensions';
```

Comment:
Should this live in the project root? It seems like it should live in a dedicated folder closer to the components, in my opinion.

Reply:
yeah I wasn't sure actually - I'll try and add it to the react-wrapper instead which lives inside `tools`. I wasn't sure if we'd want to use it for anything else, but it does seem a bit random in the root since it's web-component specific